index.pug

```diff
@@ -256,6 +250,26 @@ html(lang='en')
                     p
                       | Need to provide those information - who did when change which segment? We get you covered.
 
+ 
+    section.integration
+      .container
+        .row
+          .col-md-6
+            h2 Easy integration for your operating tools
+            p There is an integration option for all tools. Start to reduce manual processes and everday tasks with easy integration of locize for busy people. We are your cost-effective translation management platform and help move your software automation to a new level. <a href="/how-it-works.html" title="">Javascript-localization</a> made easy for apps built with React, Angular and Vue.
+          .col-md-6
+            a(href='/how-it-works.html')
+              img(src="/img/howto/i18next-frameworks.png" style="width: 80%; margin: 10px 0 0;")
+        .row
+          div(style="padding: 5rem 0 0 0")
+            .col-md-6.text-center
+              a(href='/pricing.html')
+                  img(src="/img/localization-pricing.webp" style="max-width:50%")
+            .col-md-6
+              h2 Pay only for what you use
+              p A heavy save in cost for localization management with a varying pricing. <b>Keep up to USD 100 a month in your pocket</b>  in comparison to conventional solutions with fixed prices. Check out our <a href="/pricing.html" title="localization pricing">pricing details</a> for more information. There are also more software-localization-tools available on the locize platform for free usage.
```

Comment:
- I don't know if we can say "keep up to USD 100 a month in your pocket"
- What is meant with "There are also more software-localization-tools available on the locize platform for free usage"?

Reply:
- I will rewrite 
- translation memory, review workflow, etc.?